aws-predictions/src/main/java/com/amplifyframework/predictions/aws/service/AWSPollyService.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.aws.service;
+
+import androidx.annotation.NonNull;
+
+import com.amplifyframework.core.Consumer;
+import com.amplifyframework.predictions.PredictionsException;
+import com.amplifyframework.predictions.aws.AWSPredictionsPluginConfiguration;
+import com.amplifyframework.predictions.aws.configuration.SpeechGeneratorConfiguration;
+import com.amplifyframework.predictions.aws.models.VoiceType;
+import com.amplifyframework.predictions.result.TextToSpeechResult;
+import com.amplifyframework.util.UserAgent;
+
+import com.amazonaws.AmazonClientException;
+import com.amazonaws.ClientConfiguration;
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.mobile.client.AWSMobileClient;
+import com.amazonaws.services.polly.AmazonPollyClient;
+import com.amazonaws.services.polly.AmazonPollyPresigningClient;
+import com.amazonaws.services.polly.model.OutputFormat;
+import com.amazonaws.services.polly.model.SynthesizeSpeechRequest;
+import com.amazonaws.services.polly.model.SynthesizeSpeechResult;
+import com.amazonaws.services.polly.model.TextType;
+
+import java.io.InputStream;
+
+/**
+ * Predictions service for performing text to speech conversion.
+ */
+final class AWSPollyService {
```

Comment:
I think this should be "Amazon Polly", not "AWS Polly".   And not just for Polly, for the same change should probably be made for all the other Predictions Services as well.  Maybe something to address in a separate PR.

Reply:
Our internal tech writers' page on this topic does call it "Amazon Polly". Actually, all of the services that are part of the Predictions plugins are _Amazon_ whatevers.

Reply:
Verified against our internal tech writers' page -- all of the services in this category are branded as _Amazon_ whatevers.

Reply:
shoot! I'll have to go through the code to fix those ugh

Reply:
(to be done in a separate PR as to not introduce non-polly stuff in this PR)